public/app/core/components/AppChrome/TopBar/SingleTopBar.tsx

```diff
@@ -115,11 +116,20 @@ const getStyles = (theme: GrafanaTheme2, menuDockedAndOpen: boolean) => ({
     justifyContent: 'space-between',
 
     [theme.breakpoints.up('lg')]: {
-      gridTemplateColumns: '2fr minmax(440px, 1fr)',
+      gridTemplateColumns: '2fr minmax(50%, 1fr)',
```

Comment:
i'm not sure this should be a percentage, doesn't this kind of break the `minmax`? like the min (`50%`) is always going to be higher than the max (`1fr`)